tests/framework/utils/k8s_helper.go

```diff
@@ -461,6 +461,55 @@ func (k8sh *K8sHelper) IsCRDPresent(crdName string) bool {
 	return false
 }
 
+// GetVolumeAttachmentResourceName gets the VolumeAttachment object name from the PVC
+func (k8sh *K8sHelper) GetVolumeAttachmentResourceName(namespace, pvcName string) (string, error) {
+
+	getOpts := metav1.GetOptions{}
+	pvc, err := k8sh.Clientset.PersistentVolumeClaims(namespace).Get(pvcName, getOpts)
+	if err != nil {
+		return "", err
+	}
+	return pvc.Spec.VolumeName, nil
+}
+
+//IsVolumeAttachmentResourcePresent returns true if VolumeAttachment resource is present
+func (k8sh *K8sHelper) IsVolumeAttachmentResourcePresent(namespace, volumeAttachmentName string) bool {
+
+	cmdArgs := []string{"-n", namespace, "get", "volumeattachments", volumeAttachmentName}
+	inc := 0
+	for inc < RetryLoop {
+		_, err := k8sh.Kubectl(cmdArgs...)
+		if err == nil {
+			k8slogger.Infof("Found the VolumeAttachment resource: " + volumeAttachmentName)
+			return true
+		}
+		time.Sleep(RetryInterval * time.Second)
```

Comment:
can you log what you're waiting for? If the test times out you want to know what was happening